ddp-workspace/projects/ddp-dsm-ui/src/app/FON/pages/patients-list/patients-list.component.ts

```diff
@@ -18,19 +18,19 @@ import { SearchPatientDataModel } from './models/search-patient-data.model';
 })
 
 export class PatientsListComponent implements OnInit {
+  private readonly LSParams: string = 'pListQueryParams';
```

Comment:
Can you please rename `LSParams` to something more clear to understand?

Reply:
I also thought about this. let me do it when I start refactoring that component.